common/src/types/token.rs

```diff
@@ -626,6 +626,28 @@ lazy_static! {
             ),
         ].into_iter().collect()
     };
+
+    /// Remaps Katana ERC20 addresses to their Ethereum mainnet counterparts
+    /// so that a price can be determined
+    ///
+    /// deployments.json generated by renegade-contracts
```

Comment:
nit: What does this comment mean? Is this added by mistake?

Reply:
was trying to mimic citing the source of token addresses like starknet token map:
```      
/// Remaps Starknet Goerli ERC20 addresses to their Ethereum mainnet counterparts
/// so that a price can be determined
///
/// https://github.com/starknet-io/starknet-addresses/blob/master/bridged_tokens/goerli.json
///
/// TODO: This will be removed when we implement our custom bridging solution
```
will go ahead and remove it since it's just hard coded